.github/workflows/multi-node.yml

```diff
@@ -0,0 +1,63 @@
+name: Multi node test
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags-ignore: [v.*]
```

Comment:
should multi-node tests run for each PR or only nightly?

Reply:
We have two tasks on jenkins right now, one for each master commit (`akka-multi-node`) and one for nightlies (`akka-multi-node-nightly`).